_topic_maps/_topic_map.yml

```diff
@@ -1576,6 +1576,8 @@ Topics:
     File: uninstalling-pipelines
   - Name: Creating CI/CD solutions for applications using OpenShift Pipelines
     File: creating-applications-with-cicd-pipelines
+  - Name: Using Pipelines as Code
```

Comment:
A general question that applies to the entire PR: shouldn't "Pipelines as Code" be defined and used as an attribute throughout these files?

Reply:
Done in all files, except _topic_map.yml.